packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -433,11 +433,19 @@ export const workflow = (
       );
 
       // Create a open channel objective so we can coordinate with all participants
-      await store.addObjective({
-        type: 'OpenChannel',
-        data: {targetChannelId, fundingStrategy},
-        participants: [participants[1]]
-      });
+      let data: OpenChannel['data'];
+      switch (fundingStrategy) {
+        case 'Virtual':
+          data = {fundingStrategy, targetChannelId};
+          break;
+        case 'Direct':
+          data = {fundingStrategy, targetChannelId};
+          break;
+        case 'Ledger':
+          throw new Error('Unsupported funding type');
+      }
+
+      await store.addObjective({type: 'OpenChannel', data, participants: [participants[1]]});
```

Comment:
Not created by this PR, but I wonder what is going on with `[participants[1]]`...

Reply:
I'm not sure if stored objectives really did anything in the browser wallet.